lib/fhir_models/fhir_ext/structure_definition.rb

```diff
@@ -482,18 +479,16 @@ def is_valid_json?(json)
         # consistent with the current context (element.path). For example, sometimes expressions appear to be
         # written to be evaluated within the element, other times at the resource level, or perhaps
         # elsewhere. There is no good way to determine "where" you should evaluate the expression.
-        unless element.constraint.empty?
-          element.constraint.each do |constraint|
-            if constraint.expression && !nodes.empty?
-              begin
-                result = FluentPath.evaluate(constraint.expression, json)
-                if !result && constraint.severity == 'error'
-                  @errors << "#{element.path}: FluentPath expression evaluates to false for #{name} invariant rule #{constraint.key}: #{constraint.human}"
-                end
-              rescue
-                @warnings << "#{element.path}: unable to evaluate FluentPath expression against JSON for #{name} invariant rule #{constraint.key}: #{constraint.human}"
-              end
+        next if element.constraint.empty?
```

Comment:
We can probably get rid of this; I'm pretty sure that `.each` on an empty array just doesn't execute (or, more accurately, it executes 0 times)